src/ex12_html_css_js/task_01.js

```diff
@@ -0,0 +1,10 @@
+window.onload = setElementDimensions();
```

Comment:
if you call the function in place than `window.onload = ` is unused because you assign the returned value instead of the function reference to onload event handler

Reply:
Can I something do not understand but the dimensions of the container change in accordance with the size of the browser window (see point 1)
0. Agreed. Since the script in HTML is located at the end of the page, the onload event can not be used. 
1. The setElementDimensions() function on line 1 is called to set the initial block size depending on the browser window size and is reallocated on the "resize" event"

Reply:
1. try to press `ctrl/command + "+"/"-"` when the page is loaded
2. `window.onload = setElementDimensions();` equals to  `setElementDimensions(); window.onload = undefined;`

Reply:
1. so made for better visibility of the window content. so you don't have to use scroll to read. the problem is solved by setting new min-width and min-height properties for the container block